tests/test_event.py

```diff
@@ -49,10 +51,32 @@ def test_state(event_server):
 
 
 def test_state_getter(event_server):
-    import lyrebird
     lyrebird.application.server['event'] = event_server
     test_state = lyrebird.state.get('Test')
     assert test_state == None
     lyrebird.publish('Test', 'TestMessage', state=True)
     test_state = lyrebird.state.get('Test')
     assert test_state == 'TestMessage'
+
+
+def test_customer_event_alert(event_server):
+    custom_event = CustomEventReceiver()
+    lyrebird.application.server['event'] = event_server
```

Comment:
This event server already add into "application.server"
It not necessary to add it again here.